roles/ldap/tasks/main.yml

```diff
@@ -128,24 +128,25 @@
     attributes:
       olcAccess:
         - >-
-          to dn.regex="(([^,]+),{{ services_ldap.basedn }})$"
-          by dn.exact="{{ services_ldap.binddn }}" write
-          by dn.exact=gidNumber=0+uidNumber=0,cn=peercred,cn=external,cn=auth write
-          by dn.exact,expand="cn=admin,$1" read
+          to attrs=userPassword
+          by self write
+          by anonymous auth
```

Comment:
Ik denk niet dat we dit willen: gebruiken moeten niet zelf in de ldap hun password kunnen aanpassen.  Is het niet mogelijk om de unix wachtwoorden te hergebruiken van de gebruiker (dus zonder die in de ldap te krassen, maar gewoon ldap de data uit /etc/password en shadow te laten gebruiken)?

Reply:
Dat is een copy-paste foutje denk ik en eens dat `self write` niet de bedoeling is.